### ISSUE TITLE
pgbouncer stats blank

### ISSUE THREAD

**Issue by clomdd** [darold/pgbadger#555]
On the pgbouncer tab of the pgbadger reports we have no dataset for the following:
Histogram of session times, sessions per database, sessions per user, sessions per host and most used reserved pools. Is there something that needs turning on in our logs somewhere or is this a known issue with pgbadger? Any help would be much appreciated!

**Comment by darold** [comment]
Sorry for the response delay. In your pgbouncer log file you should have lines like:
```
2019-04-11 07:10:21.827 1525 LOG Stats: 580 req/s, in 136760 b/s, out 64757 b/s,query 21449 us
```
if there is you should have a report. Which pgbadger version are you using and have you tried with latest development code?

**Comment by darold** [comment]
I guess that you was using pgbouncer >= 1.11? In this case please use latest development code it has been fixed in commit 98d4c99.
